packages/front-end/services/track.ts

```diff
@@ -80,17 +80,57 @@ const DEVICE_ID_COOKIE = "gb_device_id";
 const SESSION_ID_COOKIE = "gb_session_id";
 const pageIds: Record<string, string> = {};
 
-const dataWareHouseTrack = async (event: DataWarehouseTrackedEvent) => {
+export const dataWareHouseTrackMultipleEventsWithDefaults = async (
+  events: {
+    eventName: string;
+    properties: TrackEventProps;
+  }[]
+) => {
+  const trackedEvents = events.map((event) => ({
+    event_name: event.eventName,
+    properties_json: JSON.stringify(event.properties),
+    device_id: getOrGenerateDeviceId(),
+    page_id: getOrGeneratePageId(),
+    session_id: getOrGenerateSessionId(),
+    sdk_language: "react",
+    sdk_version: growthbook.version,
+    url: getURL(),
+    user_id: getCurrentUser()?.id,
+    context_json: JSON.stringify(growthbook.getAttributes()),
+  }));
+  await dataWareHouseTrack(trackedEvents);
+};
+
+const dataWareHouseTrackWithDefaults = async ({
+  eventName,
+  properties = {},
+}: {
+  eventName: string;
+  properties: TrackEventProps;
+}) => {
+  await dataWareHouseTrackMultipleEventsWithDefaults([
+    { eventName, properties },
+  ]);
+};
+
+const dataWareHouseTrack = async (
+  events: DataWarehouseTrackedEvent | DataWarehouseTrackedEvent[]
+) => {
   if (inTelemetryDebugMode()) {
-    console.log("Telemetry Event - ", event);
+    console.log(
+      "Telemetry Event - ",
+      Array.isArray(events)
+        ? events.map((e) => e.event_name)
+        : events.event_name
+    );
```

Comment:
We should just log `events` here.  That way we can easily inspect the full array and objects in dev tools.